roles/tendrl-server/tasks/httpd-ssl.yml

```diff
@@ -0,0 +1,101 @@
+---
+# Tasks file for Apache SSL setup for Tendrl.
+# Based on description in https://github.com/Tendrl/api/pull/264
+
+# Note: at this point, httpd package should be already installed (httpd is a
+# dependency of tendrl-api-httpd, installed along with tendrl-ui).
+
+- name: Install mod_ssl package
+  yum:
+    name=mod_ssl
+    state=latest
+
+#
+# https support over a specific IP
+#
+
+- name: Initialize new tendrl-ssl.conf file based on sample conf file
+  copy:
+    src: /etc/httpd/conf.d/tendrl-ssl.conf.sample
+    dest: /etc/httpd/conf.d/tendrl-ssl.conf
+    remote_src: True
+    # This is here to prevent overriding changes over and over again, and
+    # also to allow additional manual tweaks.
+    # If you need to start from scratch or make sure that the latest sample
+    # conf file is used, just delete the tendrl-ssl.conf file.
+    force: no
+
+# TODO: should I allow to specify servername via variable or just use
+# ansible_fqdn as I do now?
+# TODO: should I validate it here like this?
+- name: Validate ansible variables tweaking apache configuration
+  assert:
+    that:
+      - lookup('dig', ansible_fqdn) == httpd_ip_address
+
+- name: Configure VirtualHost ip address in tendrl-ssl.conf
+  lineinfile:
+    path: /etc/httpd/conf.d/tendrl-ssl.conf
+    regexp: '^<VirtualHost .*:443>'
+    line: "<VirtualHost {{ httpd_ip_address }}:443>"
+  notify:
+    - restart httpd
+
+- name: Configure VirtualHost ServerName in tendrl-ssl.conf
+  lineinfile:
+    path: /etc/httpd/conf.d/tendrl-ssl.conf
+    insertafter: '<VirtualHost .*:443>'
```

Comment:
Is there any reason not to use just ``<VirtualHost *:443>``?

Reply:
@brainfunked @anivargi see previous comment from @dahorak